modules/living_spaces_users/living_spaces_users.module

```diff
@@ -8,9 +8,43 @@
 use Drupal\Core\Access\AccessResult;
 use Drupal\Core\Cache\CacheableMetadata;
 use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Url;
 use Drupal\profile\Entity\ProfileInterface;
 
+/**
+ * Implements hook_ENTITY_TYPE_access().
+ */
+function living_spaces_users_user_access(EntityInterface $entity, $operation, AccountInterface $account) {
+  if ($operation === 'delete') {
+    if (\Drupal::moduleHandler()->moduleExists('living_spaces_intranet')) {
```

Comment:
living_spaces_intranet depends both on living_spaces_users and living_spaces_group where office_manager role is added, so why we don't add this to living_spaces_intranet module?
Also please add permission and check by permission, not role. Add permission to office_manager role after